src/collections/_documentation/development/sdk-dev/features.md

```diff
@@ -31,7 +31,7 @@ Automatic addition of useful attributes such as `tags` or `extra` or specific `c
 
 ## Breadcrumbs
 
-Manually record application events (into the current context) during the lifecycle of an application. Needs to implement a ring buffer so as not to grow indefinitely. The most recent breadcrumbs should be attached to events as they are created.
+Manually record application events (into the current scope) during the lifecycle of an application. Needs to implement a ring buffer so as not to grow indefinitely. The most recent breadcrumbs should be attached to events as they occur.
```

Comment:
Clarify who or what "Needs to implement a ring buffer" - as it reads now it is confusing.

Suggest: "Implement a ring buffer" - developers understand why to use it, I think.